app/src/layouts/DashboardLayout/AppNotificationBanner/index.tsx

```diff
@@ -41,20 +80,35 @@ export const AppNotificationBanner = () => {
   }, [newBanners?.length]);
 
   const renderAppBanner = () => {
-    const banner: Banner = newBanners ? newBanners[0] : null;
+    const banner = newBanners ? newBanners[0] : null;
 
     if (banner) {
       return (
-        <a
-          target="_blank"
-          rel="noreferrer"
-          className="app-banner"
-          href={banner?.cta ? banner.cta : null}
-          style={{ backgroundColor: banner?.backgroundColor || "000000" }}
-        >
+        <div className="app-banner" style={{ backgroundColor: banner?.backgroundColor || "#000000" }}>
+          {banner?.short_text && (
+            <span className="app-banner-badge" style={{ backgroundColor: banner?.badgeColor || "#000000" }}>
+              {banner.short_text}
+            </span>
+          )}
           <div className="app-banner-text">
-            {/* @ts-expect-error */}
-            <ReactMarkdown children={banner?.text} skipHtml={false} rehypePlugins={[rehypeRaw]} />
+            <ReactMarkdown
+              children={renderBannerText(banner.id, banner.text)}
+              skipHtml={false}
+              // @ts-ignore
```

Comment:
This does not seem necessary as we can easily mitigate the lint error by upgrading `react-markdown` to v9+
All the module upgrades will anyway be picked up soon, and this seems like an easy one to start with since the module is only being consumed here.

you can refer to [this changelog](https://github.com/remarkjs/react-markdown/releases/tag/9.0.0) (_although on a cursor reading, doesn't feel like we will have make any changes to the code to upgrade to the latest version_)